doc/userDocs/usersGuide.md

```diff
@@ -259,6 +263,118 @@ but it will not stress the parts of the network stack that manage directing traf
 from external to internal addresses.  If the performance of those components is important, 
 you should use either LoadBalancer or NodePort services.
 
+#### Configuring Namespaces<a name="namespaces"></a>
+
+Weathervane runs each instance of the Auction application in a separate namespace. The 
+workload drivers also run in their own namespace.  The default names for the application 
+namespaces are __auctionw1i*n*__, where __*n*__ is the application's instance number.  The default 
+name for the workload driver namespace is __auctionw1__.  Weathervane will try 
+to create these namespaces if they don't already exist.
+
+It is possible to customize the names for the namespaces, as well as whether Weathervane 
+should try to create the namespaces, for each Kubernetes cluster specified in your 
+configuration file. Additional details about the configuration parameters discussed 
```

Comment:
This is a great place to discuss why you might want to customize the namespaces. 

I'm wondering if we can move this section to a less prominent location in the doc. It's currently in Detailed Setup Instructions. 

Specifying namespace names and disabling namespace creation seems to be a workaround for people who have namespace restrictions on their kubernetes cluster. Is this common? We could even create a new section 'advanced customizations' or 'special cases'.

Using the prefix and suffix, however, is geared to the typical user who wants to switch between configuration sizes without reloading the database, in other words, an advanced user, so I can see how it would fit in the Detailed Setup Instructions.

This section adds a lot of length to the doc. Could we possibly try to cut down on the length of this section as well.

Reply:
I agree that this might be more detail than needed at this point in the doc.  I will move most of this content to the Advance Topics section.